src/common/components/select-repository-list/index.js

```diff
@@ -210,9 +211,9 @@ export class SelectRepositoryListComponent extends Component {
           <div className={ styles.summary }>
             { this.renderRepoAmount() }
             {'\u00A0'}
-            (<Button appearance={ 'link' } onClick={this.onHelpClick.bind(this)}>
+            (<a href="#any-missing" onClick={this.onHelpClick.bind(this)}>
```

Comment:
Vanilla doesn't have 'link' button style, so we need to get back to a regular link.